operations/system_management_health/Prometheus_Kafka_Error.md

```diff
@@ -0,0 +1,10 @@
+# Prometheus log having Prometheus-Kafka-adapter errors during installation
+
+On a fresh install of CSM 1.3.1 the Prometheus log has following example errors:
+
+```bash
+ts=2022-12-05T13:35:53.495Z caller=dedupe.go:112 component=remote level=warn remote_name=2eb187 url=http://prometheus-kafka-adapter.sma.svc.cluster.local:80/receive msg="Failed to send batch, retrying" err="Post \"http://prometheus-kafka-adapter.sma.svc.cluster.local:80/receive\": dial tcp: lookup prometheus-kafka-adapter.sma.svc.cluster.local on 10.16.0.10:53: no such host"
```

Comment:
I suggest splitting this line so a reader can see the full error text without having to scroll right.

Reply:
The error line is split so that we don't have to scroll.